Source/JavaScriptCore/bytecompiler/BytecodeGenerator.h

```diff
@@ -1290,7 +1290,17 @@ namespace JSC {
         Vector<std::pair<FunctionMetadataNode*, FunctionVariableType>> m_functionsToInitialize;
         bool m_needToInitializeArguments { false };
         RestParameterNode* m_restParameter { nullptr };
-        
+
+        struct {
+            bool needTryCatch { false };
+            FunctionNode* function { nullptr };
```

Comment:
This is the same to `m_scopeNode` of BytecodeGenerator, so this is not necessary.